### PR TITLE
Update install scripts

### PR DESCRIPTION
**Pull request by GMartinez-Sisti**

### CODE REVIEW

File: proxy/fyde_proxy_bm_vm.md [hunk 1]
```diff
@@ -1,60 +1,82 @@
...
+
+## Install script
+
+- Please note that the steps below will execute scripts obtained externally
+- We advise to inspect the content before execution
+
+1. Download and execute script
+
+    ```sh
+    curl -fsSL https://url.fyde.me/install-fyde-proxy-centos \
```
Comment (oNaiPs) [inline proxy/fyde_proxy_bm_vm.md]: @GMartinez-Sisti how about `curl -fsSL https://url.fyde.me/install-fyde-proxy-centos | sudo bash`

File: proxy/fyde_proxy_bm_vm.md [hunk 2]
```diff
@@ -1,60 +1,82 @@
...
+
+## Manual steps
 
 1. Install Software Collections (SCL) repository
 
     ```sh
-    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
-    yum -y install centos-release-scl yum-utils
-    rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-SIG-SCLo
+    sudo rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7
```
Comment (oNaiPs) [inline proxy/fyde_proxy_bm_vm.md]: @GMartinez-Sisti for those running this in docker (and copy pasting) wouldn't it be better NOT to have `sudo` here? We could require before that the user is root

File: proxy/fyde_proxy_bm_vm.md (hunk 1 above)
Comment (GMartinez-Sisti) [reply proxy/fyde_proxy_bm_vm.md]: That works, but I'm assuming the sysadmin will first want to take a look at the script before executing. WDYT?

File: proxy/fyde_proxy_bm_vm.md (hunk 2 above)
Comment (GMartinez-Sisti) [reply proxy/fyde_proxy_bm_vm.md]: In this document (proxy/fyde_proxy_bm_vm.md ) we have installation steps for Bare Metal / Virtual Machine. For docker steps we have another document -> proxy/fyde_proxy_docker.md .

File: proxy/fyde_proxy_bm_vm.md (hunk 1 above)
Comment (GMartinez-Sisti) [reply proxy/fyde_proxy_bm_vm.md]: Change it according to suggestion. Please approve again.

Comment (oNaiPs) [reply proxy/fyde_proxy_bm_vm.md]: I'd say that if they want (and they should), they'll know what to do :)

**Pull request merged**